contracts/RegistryApp.sol

```diff
@@ -40,20 +47,17 @@ contract RegistryApp is App {
      * Remove an entry from the registry.
      * @param _id The ID of the entry to remove
      */
-    function remove(
-        bytes32 _id
-    ) public auth(REMOVE_ENTRY_ROLE) {
+    function remove(bytes32 _id) isInitialized public auth(REMOVE_ENTRY_ROLE) {
         delete entries[_id];
         EntryRemoved(_id);
     }
 
     /**
-     * Get an entry from the registry.
+     * Check if an entry is in the registry.
      * @param _id The ID of the entry to get
+     * @return True if it's in the registry, false otherwise
      */
-    function get(
-        bytes32 _id
-    ) public constant returns (bytes32) {
-        return entries[_id];
+    function exists(bytes32 _id) public view returns (bool) {
```

Comment:
What's the reason for removing the ability to get the data for an entry? IMO the registry should still expose it.